src/engines/kotor2/gui/chargen/chargenname.h

```diff
@@ -28,17 +28,25 @@
 #include "src/engines/aurora/console.h"
 #include "src/engines/kotor2/gui/chargen/chargenbase.h"
 #include "src/engines/kotor2/gui/chargen/chargeninfo.h"
+#include "src/engines/kotor2/gui/widgets/label.h"
 
 namespace Engines {
 
 namespace KotOR2 {
 
+class WidgetLabel;
```

Comment:
Yeah, that doesn't work. You can't forward-declare that fake "proxy" class.

You need to either forward-declare Engines::KotOR::WidgetLabel, include "src/engines/kotor2/gui/widgets/label.h" (which you have here, but then removed again) and not forward-declare or do it inline with the definition below, i.e. "class ::Engines::KotOR::WidgetLabel *_nameLabel;"